### ISSUE TITLE
new-date doesn't work anymore on latest release 0.4.15-alpha

### ISSUE THREAD

**Issue by jgrodziski** [juxt/tick#44]
Since 0.4.15-alpha, basic constructor doesn't work anymore with IllegalArgumentException thrown when run on JVM. Just bumping the version from 0.4.14-alpha to 0.4.15-alpha throws the error.
Example:
```
(t/new-date 2019 7 17)
Execution error (IllegalArgumentException) at cljc.java-time.local-date/of (local_date.cljc:9).
no corresponding java.time method with these args
```

**Comment by henryw374** [comment]
ah. slightly over-zealous type hinting. thanks for reporting. for the mo you could `(int x)` the args, but I'll get a fix out shortly

**Comment by henryw374** [comment]
fixed in 0.4.16-alpha
